app/controllers/active_storage/representations/proxy_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+#
+# Overrides original Rails implementation to disable route:
+# /rails/active_storage/representations/proxy/:signed_blob_id/:variation_key/*filename(.:format)
+# We use "rails storage proxy" through ActiveStorage::Blobs::ProxyController
+
+class ActiveStorage::Representations::ProxyController < ActiveStorage::BaseController
+  include ActiveStorage::SetBlob
+  include ActiveStorage::SetHeaders
+
+  before_action :authorize_blob
```

Comment:
```suggestion

  include HttpAuthConcern
  include SentryConfigurationConcern
  
  before_action :authorize_blob 
  before_action :authenticate_user!
```
This may be needed. The concerns will be useful for error reporting and access control.